### ISSUE TITLE
inconsistent error counts in test report

### ISSUE THREAD

**Issue by zxkane** [aws-solutions/distributed-load-testing-on-aws#102]
**Describe the bug**

In the test result, `detail errors` is more than `error count`.

**To Reproduce**

1. create an API Gateway restful API with default limit (10000 rps)
1. create a simple HTTP endpoint test with large volume of concurrent users to load test the API Gateway endpoint exceeding the default 10000 rps, for example, 30 tasks with 15 concurrency
1. the rps of HTTP API will exceed the limit 10000, the exceeded requests would be throttled by API Gateway with **429** status code 

**Expected behavior**

In my test result, the `Error Count` is `703317`, however `1383250` `429 error` reported in `Errors`. See below screenshot for detail,

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/843303/194766302-079c62f6-47da-4602-929d-c2ca140844fb.png">

Also the documentation lacks detail description for `Errors`, it would be helpful adding some examples as well.

https://docs.aws.amazon.com/solutions/latest/distributed-load-testing-on-aws/test-results.html

**Please complete the following information about the solution:**
- [x] Version: v3.0.0
- [x] Region: ap-northeast-2
- [x] Was the solution modified from the version published on this repository? no
- [ ] If the answer to the previous question was yes, are the changes available on GitHub?
- [x] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the services this solution uses?
- [ ] Were there any errors in the CloudWatch Logs?

**Screenshots**

**Additional context**

**Comment by G-Lenz** [comment]
Hi @zxkane 
Thank you for pointing this out. We have added this to our backlog and will be investigating this further.

**Comment by cshowersPersonal** [comment]
Seeing this as well, every run.

**Comment by kamyarz-aws** [comment]
strange! I will try to recreate the issue on our end. Most of our numbers  are being directly extracted from jmeter/taurus and we dont manipulate them for reporting. I have to see if I can generate this on my end, and if there was an error on our end I will try to address it if not, I have to create a ticket with jmeter to address it.

**Comment by cshowersPersonal** [comment]
Example, started happening about 1.5-2 months ago.  I haven't changed JMeter and recently updated DLT.  Suspect taurus 
![image (1)](https://github.com/aws-solutions/distributed-load-testing-on-aws/assets/145483468/d046121d-eb32-44a5-9075-4ec1503fcb92)

**Comment by kamyarz-aws** [comment]
Yeah my guess was taurus basically. Thanks for calrifying this though.
